src/bindings.mli

```diff
@@ -0,0 +1,25 @@
+open Stdune
```

Comment:
Could you add a comment describing what this module is about? It might be hard to guess for someone new to the codebase.